OpenRA.Mods.Common/Activities/Air/Land.cs

```diff
@@ -19,18 +19,27 @@ public class Land : Activity
 	{
 		readonly Target target;
 		readonly Aircraft aircraft;
-		readonly bool requireSpace;
 		readonly Actor ignoreActor;
 
 		bool landingInitiated;
 		bool soundPlayed;
 
-		public Land(Actor self, Target t, bool requireSpace, Actor ignoreActor = null)
+		public Land(Actor self, Target t, Actor ignoreActor = null)
```

Comment:
Can we please use this opportunity to get rid of the hacky `ignoreActor` parameter too?

The simplest way is to replace `ignoreActor` with a `WVec offset` and then change `ReturnToBase` to call this as QueueChild(self, new Land(self, Target.FromActor(dest), offset), true);`.

IMO a better way is to pass the `Exit` itself, and treat it as the requested docking point -- this takes us another step towards docking logic and airfields that support multiple actors.

Reply:
> Maybe best to do the simple approach here, and then migrate to `Exit` in a future PR?

As I've mentioned before, I'm working on (or rather have pretty much finished and just wait for dependencies to get merged) a refactor of `Reservable` into a `Dock` trait that should be more suitable than `Exit`, so yeah, I'd like to leave the migration to a follow-up.